src/openforms/appointments/exceptions.py

```diff
@@ -8,3 +14,9 @@ class AppointmentDeleteFailed(AppointmentException):
 
 class AppointmentCreateFailed(AppointmentException):
     pass
+
+
+class CancelAppointmentFailed(APIException):
+    status_code = status.HTTP_400_BAD_REQUEST
```

Comment:
It doesn't seem like this is a _client_ error, which 4xx indicates. The client did nothing wrong.

I would rather expect HTTP 503 here or perhaps HTTP 502

@joeribekker any preference/input?

Reply:
For the moment I've made this a 502.